ssr/src/utils/token/icpump.rs

```diff
@@ -0,0 +1,54 @@
+use serde::{Deserialize, Serialize};
+
+use futures::stream::BoxStream;
+use futures::StreamExt;
+
+use leptos::*;
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct TokenListItem {
+    pub user_id: String,
+    pub name: String,
+    pub token_name: String,
+    pub token_symbol: String,
+    pub logo: String,
+    pub description: String,
+    pub created_at: String,
```

Comment:
can we store the epoch milliseconds in the firestore db instead? I want to avoid adding chrono as a dependency

Reply:
Another option is: https://crates.io/crates/speedate
Which is far lightweight compared to chrono